queries.sql

```diff
@@ -94,3 +94,34 @@ SELECT species, AVG(escape_attempts) AS average_escape_attempts
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
 GROUP BY species;
+
+-- Write queries using JOIN
+
+BEGIN;
+
+-- Update owner_id for Agumon
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith')
+WHERE name = 'Agumon';
+
+-- Update owner_id for Gabumon and Pikachu
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell')
+WHERE name IN ('Gabumon', 'Pikachu');
+
+-- Update owner_id for Devimon and Plantmon
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob')
+WHERE name IN ('Devimon', 'Plantmon');
+
+-- Update owner_id for Charmander, Squirtle, and Blossom
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond')
+WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
+
+-- Update owner_id for Angemon and Boarmon
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
+WHERE name IN ('Angemon', 'Boarmon');
+
+COMMIT;
```

Comment:
- [ ] Good job so far @Roland-Ntwali There are just a few changes you need to make and you'll be on your way to the next Milestone. Please note that these contents are supposed to be in the `data.sql` file and not this file since they are not queries instead they are a way of modifying the animal's table to meet the project requirements. So kindly include this in the `data.sql` file instead. :+1: 
- [ ] I have attached the project requirements below as well :100: 
- Project Requirements
![image](https://github.com/Roland-Ntwali/Vet-clinic/assets/102699958/122d56f0-67c9-4db2-a49b-237c80c77c8a)